docs/game_sdk/Networking.md

```diff
@@ -14,9 +14,9 @@ Need a networking layer? Have a networking layer! This manager handles all thing
 
 An important note to make here is that our networking layer **is not peer-to-peer**. Discord has always promised that we will not leak your IP, and we promise to keep it that way. Though it seems like you are connected directly to another user, it routes through Discord's servers in the middle, ensuring both security and robust networking thanks to our servers.
 
-## GetSessionId
+## GetPeerId
 
-Get the unique networking session id for the current user. Each user has a unique id for that session, used for connecting to that user.
+Get the unique networking peerId for the current user. Each user has a unique id for that session, used for connecting to that user.
```

Comment:
I'd call it "peer ID", not "peerId", when talking about is as a concept (i.e. not a variable or function name).

Reply:
"that session" is ambiguous now that we're not talking about a session ID.